pytorch_blade/torch_blade/onnx_backends/backend_conversion.py

```diff
@@ -37,7 +39,21 @@ def _deduplicate_onnx_graph_outputs(graph):
         visited.add(v)
 
 
+def _try_cast_graph_integer_inputs_to_i32(graph):
+    # Refer to https://github.com/onnx/onnx-tensorrt/blob/main/docs/operators.md.
+    # There is limited support for INT32, INT64, and DOUBLE types
+    cfg = Config.get_current_context_or_new()
+    need_fixup_backends = ["TensorRT"]
+    if cfg.optimization_pipeline not in need_fixup_backends:
+        return
+    # Convert INT64 to INT32.
+    for val in graph.inputs():
+        if val.type().scalarType() == "Long":
+            tools.cast_to_i32_tensor_type(val)
```

Comment:
Where does this `cast_to_i32_tensor_type` function come from? I searched our codebase but couldn't see its definition. I'm wondering how could we fix issue at build time. Does it mutate the graph?

Reply:
I had missed adding it.